app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -18,23 +18,6 @@ module WorksControllerBehavior
       rescue_from WorkflowAuthorizationException, with: :render_unavailable
     end
 
-    class_methods do
-      def cancan_resource_class
-        Hyrax::ControllerResource
-      end
-    end
-
-    def build_change_set(resource)
-      change_set_class.new(resource,
-                           depositor: current_user.user_key,
```

Comment:
Depositor isn't in the `ResourceController` equivalent.  Should it be?

Reply:
Good eye. I've restored this (but it's moved to the private section)